dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -439,14 +442,14 @@ def _merge_biased_kurtosis(match_count1, biased_kurtosis1, biased_skewness1,
         :return: combined skewness
         :rtype: float
         """
-        if np.isnan(biased_kurtosis1):
-            biased_kurtosis1 = 0
-        if np.isnan(biased_kurtosis2):
-            biased_kurtosis1 = 0
         if match_count1 < 1:
             return biased_kurtosis2
         elif match_count2 < 1:
             return biased_kurtosis1
+        if np.isnan(biased_kurtosis1):
+            biased_kurtosis1 = 0
+        if np.isnan(biased_kurtosis2):
+            biased_kurtosis1 = 0
```

Comment:
if any value nan that has match_count > 1, should we instead just return nan? In that case could this be an elif part of the above?

Reply:
Agreed, changes made